keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_presets.py

```diff
@@ -0,0 +1,778 @@
+    "efficientnetv2-s": {
+        "metadata": {
+            "description": "The EfficientNet small architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(SMALL_BLOCK_ARGS)} convolutional blocks."
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.0,
+            "default_size": 384,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": SMALL_BLOCK_ARGS,
+        },
+    },
+    "efficientnetv2-m": {
+        "metadata": {
+            "description": "The EfficientNet medium architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(MEDIUM_BLOCK_ARGS)} convolutional blocks."
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.0,
+            "default_size": 480,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": MEDIUM_BLOCK_ARGS,
+        },
+    },
+    "efficientnetv2-l": {
+        "metadata": {
+            "description": "The EfficientNet medium architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(LARGE_BLOCK_ARGS)} convolutional blocks."
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.0,
+            "default_size": 480,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": LARGE_BLOCK_ARGS,
+        },
+    },
+    "efficientnetv2-b0": {
+        "metadata": {
+            "description": "The EfficientNet B0 architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(B0_BLOCK_ARGS)} convolutional blocks. As with all of the B "
+            "style EfficientNet variants, the number of filters in each "
+            "convolutional block is scaled by a `width_coefficient` and a "
+            "`depth_coefficient`.  Please see the GitHub source to find the "
```

Comment:
Why not just write the coefficients here? (same throughout the file)

Reply:
Great point - updated